.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,197 @@
+name: Build wheels
+
+on:
+  # Run daily at 1:23 UTC
+  schedule:
+    - cron: '23 1 * * *'
```

Comment:
Run nightly to build wheels for the nightly upload. If people think a nighlty cadance is too excessive we can make this weekly.

Reply:
I think PRs get merged once every few days, so somewhere between daily and weekly is the optimum. Daily is fine for now.